sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala

```diff
@@ -397,15 +398,15 @@ class DateTimeUtilsSuite extends SparkFunSuite with Matchers with SQLHelper {
 
     // transit from Pacific Standard Time to Pacific Daylight Time
     assert(timestampAddInterval(
-      ts1, 0, 0, 23 * CalendarInterval.MICROS_PER_HOUR, TimeZonePST.toZoneId) === ts2)
+      ts1, 0, 0, 23 * IntervalConstants.MICROS_PER_HOUR, TimeZonePST.toZoneId) === ts2)
```

Comment:
We don't need the prefix `IntervalConstants.` cuz we've already imported `xxx.IntervalConstants._`.